### PR TITLE
Example FrEDI Rmd

### PR DESCRIPTION
**Pull request by emcduffie**
Adding new folder with an Example FrEDI run Rmd that formats, runs, and plots FrEDI data based on SSP emission scenarios

### CODE REVIEW

File: examples/basic_run_example.Rmd [hunk 1]
```diff
@@ -0,0 +1,619 @@
+---
+title: "FrEDI Run Example"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, eval=TRUE)
+
+# USERS NEED TO SPECIFY THIER WORKING DIRECTORY AND PYTHON PATHS
```
Comment (cahartin) [inline examples/basic_run_example.Rmd:9]: I'm wondering if we can pull this out into 2 different Rmd or not include FaIR in this write up. Part of me wants to keep it focused and narrow on FrEDI.

File: examples/basic_run_example.Rmd [hunk 2]
```diff
@@ -0,0 +1,619 @@
...
+# NOTE: it will take a long time (~minutes to hours) to load the required FrEDI packages if not
+# already installed on your local machine
+
+# Specify if you want to install the FrEDI package (TRUE = Y, FALSE = N)
+# If already installed, set to FALSE
+inst_flag = FALSE
+
+#Specify branch
+branch = 'main'
+
```
Comment (cahartin) [inline examples/basic_run_example.Rmd:80]: add comment here saying that you can also load a different branch or tagged release

File: examples/basic_run_example.Rmd [hunk 3]
```diff
@@ -0,0 +1,619 @@
...
+      ref = branch
+    ))
+  library("FrEDI")
+} else {
+  require("FrEDI")
+}
+```
+
+
+## Step 2. Use FaIR to create a temperature input for FrEDI (optional)
```
Comment (cahartin) [inline examples/basic_run_example.Rmd:99]: i'm leaning towards deleting the FaIR part.....i think it takes away from FrEDI.

File: examples/basic_run_example.Rmd [hunk 4]
```diff
@@ -0,0 +1,619 @@
...
+## FaIR is currently run with default parameters. Please see the FaIR GitHub 
+## repository for additional information on FaIR parameters. 
+
+
+import pandas as pd
+import numpy as np
+import fair as py_fair
+print('Using FaIR version: ',py_fair.__version__)
+
+#import the SSP emissions from FaIR repo
```
Comment (cahartin) [inline examples/basic_run_example.Rmd:130]: if we do keep this, then lets simplify and only include 1 SSP

File: examples/basic_run_example.Rmd [hunk 5]
```diff
@@ -0,0 +1,619 @@
...
+
+# Select temperature file 
+# Choose to use fair or pre-processed files
+# un-comment the desired scenario
+
+use_calcd_fair = F
+
+if (use_calcd_fair ==T){
+
+  # FaIR output from Step 2
```
Comment (cahartin) [inline examples/basic_run_example.Rmd:184]: let's provide 1 output for this Rmd

File: examples/basic_run_example.Rmd [hunk 6]
```diff
@@ -0,0 +1,619 @@
...
+# Note: to run more than one scenario, the code below 
+# must be adapted into a loop to be able to format 
+# the inputs and run FrEDI for each scenario. 
+
+#***********************************************
+#1. Specify Input Files and Parameters
+
+## Input Files
+tempInputFile <- temp_file_loc
+  # csv file with time series of temperature relative to 1995 (units: degC)
```
Comment (cahartin) [inline examples/basic_run_example.Rmd:221]: relative to 1986-2005

File: examples/basic_run_example.Rmd [hunk 7]
```diff
@@ -0,0 +1,619 @@
...
+# must be adapted into a loop to be able to format 
+# the inputs and run FrEDI for each scenario. 
+
+#***********************************************
+#1. Specify Input Files and Parameters
+
+## Input Files
+tempInputFile <- temp_file_loc
+  # csv file with time series of temperature relative to 1995 (units: degC) 
+  #   (values: >=0 and <= 10)
```
Comment (cahartin) [inline examples/basic_run_example.Rmd:222]: values can be greater than 10. i would just specify that temps have to be >0

File: examples/basic_run_example.Rmd [hunk 8]
```diff
@@ -0,0 +1,619 @@
...
+## Step 4. Plot
+
+The following chunks provide examples of figures generated from FaIR and FrEDI output
+
+#### Define plotting parameters
+Use this chunk to define sector categories and global plotting parameters
+```{r plot_params}
+#Define parameters
+
+#define aggregate categories
```
Comment (cahartin) [inline examples/basic_run_example.Rmd:364]: sample sector aggregations similar to Hartin et al., 2023

File: examples/basic_run_example.Rmd [hunk 9]
```diff
@@ -0,0 +1,619 @@
...
+
+#Define color palette
+colors_6 = c("#f2938c", "#d54309", "#ffbc78", "#4d8055", "#97d4ea",
+             "#1a4480")
+colors_7 = c("#f2938c", "#d54309", "#ffbc78", "#86b98e","#4d8055",
+             "#97d4ea","#1a4480")
+```
+
+
+#### Plot Temperature Trajectory
```
Comment (cahartin) [inline examples/basic_run_example.Rmd:392]: i'd be inclined to leave out. users can plot up own temps

File: examples/basic_run_example.Rmd [hunk 10]
```diff
@@ -0,0 +1,619 @@
...
+
+```{r, print_tables, echo = FALSE, results = 'asis'}
+
+# print table of total annual national damages by sector 
+# for a specific year
+
+c_year = 2090
+
+#Table 1
+table1 <- output_df %>%
```
Comment (cahartin) [inline examples/basic_run_example.Rmd:534]: is table1 the same as df_filter? if so, let's simplify and call df_filter

**Review by cahartin — CHANGES_REQUESTED** [review]

**Review by emcduffie — COMMENTED** [review]
@cahartin - I addressed the issues we discussed today. Take a look at the new FrEDI_run_example.Rmd

**Comment by cahartin** [comment]
@emcduffie do we still want this if we are setting up a pages instead?

**Comment by emcduffie** [comment]
We might. I need to play around with pages a little more to figure out what makes the most sense.

**Pull request closed without merging**